backend/django/core/views/api_admin.py

```diff
@@ -40,18 +40,34 @@ def label_distribution(request, project_pk):
     users.append(project.creator)
     users.extend([perm.profile for perm in project.projectpermissions_set.all()])
 
+    def sortCount(label):
+        label_count = 0
+        for u in users:
+            label_count += DataLabel.objects.filter(profile=u, label=label).count()
+        return label_count
+
+    # Sort labels by the count
+    labels.sort(key=sortCount, reverse=True)
```

Comment:
I think the goal of the sortCount function and this line is to sort the label list by the total number of datalabels right? I think you can simplify this to one line

`labels.sort(key=lambda label: DataLabel.objects.filter(label=label).count(), reverse=True)`